packages/reports/addon/templates/components/navi-date-range-picker.hbs

```diff
@@ -1,5 +1,5 @@
-{{!-- Copyright 2017, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
-{{#pick-container updateSelection=this.attrs.setInterval as |selection container|}}
+{{!-- Copyright 2019, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+{{#pick-container onUpdateSelection=this.attrs.onSetInterval as |selection container|}}
```

Comment:
Since we are touching this, can we remove all the `this.attrs`? It is now a private API